setup.py

```diff
@@ -56,7 +56,7 @@
         "lru-dict>=1.1.6",
         # When updating to a newer version of pyrsistent, please check that the interface
         # `transform` expects has not changed (see https://github.com/tobgu/pyrsistent/issues/180)
-        "pyrsistent>=0.16.0,<0.17",
```

Comment:
Could we keep this as `>=0.16.0` or does it break tests?

Reply:
It doesn't break anything when installing with a supported python version, because it'll never be installed. I've just been going with the lowest version that supports our lowest version of python, and 0.16.0 stops at py37 - https://pypi.org/project/pyrsistent/0.16.1/

Reply:
Thinking about this more, I think I'd prefer to keep it at `>=0.16.0` so if someone is using this with an older version of python they still can. Unless we have other breaking changes queued up too?

Reply:
Leaving at `>=0.16.0`. Next breaking will likely be dropping py38, but shouldn't hurt anything.